PRChecker/macOS/AppDelegate.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AppDelegate.swift
+//  PRChecker
+//
+//  Created by Chen Yuhan on 2021/11/07.
+//
+
+import Cocoa
+
+class AppDelegate: NSObject, NSApplicationDelegate {
+
+    private var statusItem: NSStatusItem?
+    private var menuController: MenuController?
+
+    func applicationDidFinishLaunching(_ aNotification: Notification) {
+        statusItem = NSStatusBar.system.statusItem(withLength: 16)
+
+        guard let statusItemButton = statusItem?.button else { return }
+
+        statusItemButton.image = NSImage(
+            systemSymbolName: "drop",
```

Comment:
Any deep meaning behind the choice of drop?

Reply:
No reason 😂, just because `Drop` has a beautiful shape